modules/nf-core/samblaster/environment.yml

```diff
@@ -0,0 +1,7 @@
+channels:
+  - conda-forge
+  - bioconda
+  - defaults
+dependencies:
+  - bioconda::samblaster=0.1.26
+  - bioconda::samtools=1.16.1
```

Comment:
we should update samtools here

Reply:
https://github.com/nf-core/modules/issues/4276

Reply:
Merge in?

Reply:
that's just an issue